airframe-http-recorder/src/main/scala/wvlet/airframe/http/recorder/HttpRecord.scala

```diff
@@ -43,13 +43,23 @@ case class HttpRecord(session: String,
   }
 
   def toResponse: Response = {
-    val r        = Response(Version.Http11, Status.fromCode(responseCode))
-    val rawBytes = responseBody.getBytes(StandardCharsets.UTF_8)
-    r.content = Buf.ByteArray.Owned(rawBytes)
-    r.contentLength = rawBytes.length
+    val r = Response(Version.Http11, Status.fromCode(responseCode))
+
     responseHeader.foreach { x =>
       r.headerMap.set(x._1, x._2)
     }
+
+    val contentBytes = responseHeader.find(h => h._1.equalsIgnoreCase("content-type")) match {
+      case Some(x) if x._2 == MediaType.OctetStream =>
```

Comment:
Supporting only octet-stream is OK? Rather, treating all responses as binary might be better?